content/develop/concepts/multipage-apps/page-and-navigation.md

```diff
@@ -0,0 +1,75 @@
+---
+title: Define multipage apps with st.Page and st.navigation
+slug: /develop/concepts/multipage-apps/page-and-navigation
+description: Understand the most flexible and preferred method for defining multipage apps
+---
+
+# Define multipage apps with `st.Page` and `st.navigation`
+
+`st.Page` and `st.navigation` are the preferred method for defining multipage apps. With these functions, you have flexibility to organize your project files and customize your navigation menu. Simply initialize `StreamlitPage` objects with `st.Page`, then pass those `StreamlitPage` objects to `st.navigation` in your entrypoint file (i.e. the file you pass to `streamlit run`).
```

Comment:
Thoughts on adding a simple code example nearer to the top so someone gets the gist of where it's going?

(Just an idea, not a strong preference)

Reply:
> `st.Page` and `st.navigation` are the preferred method for defining multipage apps. With these functions,

Use of function/method interchangeably. Suggestion: [`commands`](https://www.notion.so/snowflake-corp/Streamlit-brand-style-guide-b05db4418b44480ca50ba36e906bb3f9?pvs=4#b2d41747627e4bf1a4577dc90e0b2983)

Reply:
"method" as in "manner" not "function" in this case, but fair point.

Reply:
I'll change it to commands, but it feels like it's implying there are other (command) alternatives when there's not.